tests/apollo/util/pyclient/bft_client.py

```diff
@@ -47,6 +67,9 @@ def next(self):
         Calculate the next req_seq_num.
         Return the calculated value as an int sized for 64 bits
         """
+
+        globals()['__COUNT'].inc()
+        return self.val()
         milli = int(time.time() * 1000)
```

Comment:
If the old use to get a sequence number is not going to be used anymore, please either remove it or implement them both (in two different ReqSeqNum implementations) and let the user choose which one he wants with a flag

Reply:
done